src/main/java/com/teamcautionrobotics/robot/DriveBase.java

```diff
@@ -29,6 +30,19 @@
     public PIDController pidController;
     public final DriveBasePIDOutput pidOutput;
 
+    private double lastLeftPower;
+    private double leftInputDerivative;
+
+    private double lastRightPower;
+    private double rightInputDerivative;
+
+    // Change in time
+    private final double dt = TimedRobot.kDefaultPeriod;
+
+    // This value is the derivative of the input power, which is only proportional
+    // to the actual jerk of the robot in m/s^3
+    private double jerkLimit = 1;
```

Comment:
How was this number chosen, or was it completely arbitrary?

Reply:
It was completely arbitrary.